include/nicegraf-wrappers.h

```diff
@@ -374,6 +374,57 @@ class xfer_encoder {
   ngf_xfer_encoder enc_;
 };
 
+/**
+ * \ingroup ngf_wrappers
+ *
+ * Wraps a compute encoder with unique ownership semantics.
+ */
+class compute_encoder {
+  public:
+  /**
+   * Creates a new compute encoder for the given command buffer. Has the same semantics as \ref
+   * ngf_cmd_begin_compute_pass.
+   *
+   * @param cmd_buf The command buffer to create a new compute encoder for.
+   */
+  explicit compute_encoder(
```

Comment:
while we're here, can we also add ctors _without_ `sync_op` parameter that just pass `nullptr` to their respective `begin_xxx_pass` commands? here and in other encoder wrappers above. thx 😊

Reply:
> while we're here, can we also add ctors _without_ `sync_op` parameter that just pass `nullptr` to their respective `begin_xxx_pass` commands? here and in other encoder wrappers above. thx 😊

done ^^

Reply:
added comments for ctors without sync_op